OpenAI_API/ChatFunctions/Function_Call.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace OpenAI_API.ChatFunctions
+{
+    /// <summary>
+    /// An optional class to be used with models that support returning function calls.
+    /// </summary>
+    public class Function_Call
```

Comment:
You would also want to rename this file to FunctionCall.cs
```suggestion
    public class FunctionCall
```